torchbench.py

```diff
@@ -730,18 +776,43 @@ def main():
 
     experiment = functools.partial(experiment, args, model_iter_fn)
 
+    if args.speedup_fx2trt_fp16:
+        cos_similarity = True
+    else:
+        cos_similarity = False
+
     if output_filename:
         output_filename = os.path.join(torchdynamo.config.base_dir, output_filename)
 
     if args.minimum_call_count:
         torchdynamo.config.minimum_call_count = args.minimum_call_count
-
     if args.only:
         for device in args.devices:
             try:
                 device, name, model, example_inputs = load_model(
                     device, args.only, args.training, args.check_accuracy
                 )
+                # torchbench changed the default precison=fp16 on torchvision net
+                if name in (
+                    "alexnet",
+                    "resnet18",
+                    "resnet50",
+                    "mobilenet_v2",
+                    "mnasnet1_0",
+                    "squeezenet1_1",
+                    "shufflenetv2_x1_0",
+                    "vgg16",
+                    "resnext50_32x4d",
+                ):
+                    try:
+                        assert (
+                            args.speedup_fx2trt_fp16 == True
```

Comment:
Can we move this to `speedup_experiment_fx2trt()`, you can use the global variable `current_name`.  I believe this disables these models for everything, not just fx2trt.

Reply:
Thanks for pointing that out. 
Hm, we can not put it into speedup_experiment_fx2trt. Since we need to stop it earlier than that. If not, the error will throw in line 923: 
elif not same(correct_result, new_result, cos_similarity):

Reply:
Could it go inside the `if` on L813?  Just want to make sure this doesn't change things when not using fx2trt.

Reply:
Got it. Maybe it is better to have another `if args.speedup_fx2trt: ` and put them under this if